code/src/nuvla/connector/docker_compose_cli_connector.py

```diff
@@ -84,11 +84,15 @@ def start(self, **kwargs):
                 config.close()
                 env['DOCKER_CONFIG'] = tmp_dir_name
 
+            cmd_pull = self.build_cmd_line(
+                ['-p', project_name, '-f', compose_file_path, 'pull'])
+
             cmd_deploy = self.build_cmd_line(
                 ['-p', project_name, '-f', compose_file_path, 'up', '-d',
                  '--remove-orphans'])
 
-            result = self._execute_clean_command(cmd_deploy, env=env)
+            result = self._execute_clean_command(cmd_pull, env=env)
+            result += self._execute_clean_command(cmd_deploy, env=env)
```

Comment:
doesn't this var get cluttered with information? Can you paste an example of how it looks like?

Reply:
Which var ? `result` ?
If yes, it contain the output of the `docker-compose pull` command. 
The output is similar to the output of `docker-compose up`.
Basically a list of images and their pull result.
For example: 
```
Pulling system-manager               ... done
Pulling compute-api                  ... done
Pulling agent                        ... failed
```

Reply:
but you are concatenating both `pull` and `up`. So, is the final string easy to ready? Isn't it too big to post as the job_status?

Reply:
I think it's ok, it depends on the number of containers and images of the app.
At the very very most it will double the size of the message.
Should be fine for the job_status.